MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,775 @@
+---
+layout: post
+title: Working with DataForm in .NET MAUI DataForm control | Syncfusion
+description: Learn about the working with DataForm in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with DataForm in Maui DataForm (SfDataForm)
+
+## Auto-generating DataFormItems for the data field
+
+By default, the `SfDataForm.Items` will be generated based on the property type. For example, the `DataFormTextItem` will be created for the `string` type property. 
```

Comment:
will be generated based on the property type in 'DataObject`

Reply:
content updated